src/wasm-traversal.h

```diff
@@ -256,18 +256,18 @@ struct Walker : public VisitorType {
         self->walkTag(curr.get());
       }
     }
-    for (auto& curr : module->tables) {
-      self->walkTable(curr.get());
-    }
     for (auto& curr : module->elementSegments) {
       self->walkElementSegment(curr.get());
     }
-    for (auto& curr : module->memories) {
-      self->walkMemory(curr.get());
+    for (auto& curr : module->tables) {
+      self->walkTable(curr.get());
     }
     for (auto& curr : module->dataSegments) {
       self->walkDataSegment(curr.get());
     }
+    for (auto& curr : module->memories) {
+      self->walkMemory(curr.get());
+    }
```

Comment:
This change only seems ok because I can't think of any other pass that would notice it... but in theory there could be. I guess if tests pass it's fine.